packages/opentelemetry-core/src/platform/browser/constants.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020, OpenTelemetry Authors
```

Comment:
Maybe we can rename file to move specific name (ex: sdk-info.ts) instead of constants ,